pkg/bootstrap/bootstrap_config.go

```diff
@@ -231,28 +231,82 @@ func extractMetadata(envs []string, prefix string, set setMetaFunc, meta map[str
 	}
 }
 
+type istioMetadata struct {
+	IP             string            `json:"ip"`
+	Labels         map[string]string `json:"labels"`
+	Name           string            `json:"name"`
+	Namespace      string            `json:"namespace"`
```

Comment:
is deployment, replica set part of labels or should they be separate items here?

Reply:
Plan is to add that stuff incrementally. So, when we get to the sidecar injector portion, we can consider moving this struct into a shared location and add the appropriate fields.